be/src/Project/Services/Identity/Identity.Api/Areas/Accounts/Resources/AccountResources.pl.resx

```diff
@@ -156,6 +159,12 @@
   <data name="LastNameRequiredErrorMessage" xml:space="preserve">
     <value>Wprowadź swoje nazwisko</value>
   </data>
+  <data name="MarketingApprovalHeader" xml:space="preserve">
+    <value>Formularz marketingowy dla przedsiębiorców współpracujących z ELTAP</value>
+  </data>
+  <data name="MarketingApprovalText" xml:space="preserve">
+    <value>W przypadku chęci otrzymywania od nas informacji o aktualnej ofercie, obowiązujących kampaniach, wprowadzonych funkcjonalnościach czy zmianach, a także innych działaniach prowadzonych przez ELTAP, prosimy o wypełnienie poniższych zgód marketingowych. Udzielone zgody można wycofać w każdym czasie, przy czym cofnięcie zgody nie wpływa na działania, które zostały podjęte przed dniem jej cofnięcia.</value>
```

Comment:
**suggestion:** Word ELTAP couldn't be in text, because this app is not used only by ELTAP